datagrepper/app.py

```diff
@@ -776,3 +781,14 @@ def internal_error(error):
         status=500,
         mimetype="application/json",
     )
+
+
+def liveness():
+    pass
+
+
+def readiness():
+    try:
+        datagrepper.ping()
```

Comment:
Where does this "ping" function come from? I can't find it.

Reply:
Just so I understand this better, in pseudo-code it would look something like this - 

def readiness():
       try:
             connect_to_the_database()
             query = "SELECT 1 FROM datagrepper;"
       except Exception:
              raise HealthError("Can't connect to the database")      
      return (query)

Reply:
Yeah, something like that. The DB connection should already have been established, it is done at the start of the application (see `dm.init()` in `app.py`). (it is, by the way, not the best way to do it, but that's a different issue)

What you have to do in this function is make sure that you can actually query the database by executing the SQL query.
Also, the query can just be `SELECT 1`, no need for the `FROM tablename`, we're just trying to check that the DB server is responding here, not running actual data queries.
And you don't need to return the query.

Reply:
Have to fix some of the tests, but the code should be closer to what we want